dags/public-health/covid19/sync_covid_testing_data.py

```diff
@@ -0,0 +1,99 @@
+"""
+Pull data from MOPS COVID Dashboard and upload to ESRI
+"""
+import datetime
+import os
+
+import arcgis
+import pandas as pd
+
+from airflow import DAG
+from airflow.hooks.base_hook import BaseHook
+from airflow.hooks.S3_hook import S3Hook
+from airflow.operators.python_operator import PythonOperator
+
+
+def get_data(filename, workbook, sheet_name):
+    df = pd.read_excel(workbook, sheet_name=sheet_name, skiprows=1, index_col=0)
+    df = df.T
+    df = df.iloc[:, 1:3]
+    df.reset_index(level=0, inplace=True)
+    df.columns = ["Date", "Performed", "Test Kit Inventory"]
+    df["Date"] = pd.to_datetime(df["Date"], errors="coerce")
+    df = df.loc[df["Date"].dt.date < datetime.datetime.now().date()]
```

Comment:
datetime.now would probably also need to become timezone-aware.

Reply:
See previous comment